common/src/main/resources/db/changelog/v2021/add_synthea_contracts.sql

```diff
@@ -32,9 +34,10 @@ insert into user_role(user_account_id, role_id) values ((select id from user_acc
 insert into user_role(user_account_id, role_id) values ((select id from user_account where client_id='0oa9jyzerqYlfXtsQ297' limit 1), (select id from role where name='SPONSOR' limit 1));
 insert into user_role(user_account_id, role_id) values ((select id from user_account where client_id='0oa9jyy4ou5c1MJ3N297' limit 1), (select id from role where name='SPONSOR' limit 1));
 insert into user_role(user_account_id, role_id) values ((select id from user_account where client_id='0oa9jyw49f9U19vvB297' limit 1), (select id from role where name='SPONSOR' limit 1));
+insert into user_role(user_account_id, role_id) values ((select id from user_account where client_id='0oa9jyw49f9U20vvB297' limit 1), (select id from role where name='ADMIN' limit 1));
```

Comment:
Setting this new line's role name to `ADMIN` gives us test coverage on the `return false` case inside of the [PdpClientServiceImpl.hasSponsorRole](https://github.com/CMSgov/ab2d/blob/3e23a031c72373523c4e79191b69f35604642b0d/common/src/main/java/gov/cms/ab2d/common/service/PdpClientServiceImpl.java#L53-L61) function.

The problem is... I don't know where this data is loaded, or how it is supposed to be used. It is only test data? If so, then why isn't it inside of the test folder? Is it perhaps sandbox data? If so, we me adding fake data in here possibly break something in sandbox? I unfortunately really don't know much about this system, so I can't project the impact of this change.

All I know, is that adding this line gives us more test coverage. And its the only way to get test coverage on that line, as far as I can tell. So here I am, trying to figure out if this change is safe or not 🤷🏼